src/containers/Questions/index.js

```diff
@@ -1,8 +1,122 @@
 import React from 'react';
-import UnderConstruction from '../UnderConstruction';
+import propTypes from 'prop-types';
+import { message } from 'antd';
+import Question from './questions';
 
-const Questions = props => {
-  return <UnderConstruction {...props} />;
+import schema from './questionValidation';
+import entryData from './data';
+
+class Questions extends React.Component {
+  state = {
+    current: 0,
+    title: '',
+    content: '',
+    errors: {},
+    journals: [],
+  };
+
+  handleConfirm = e => {
+    message.warning("You didn't make an entry today");
+    const { history } = this.props;
+    history.push('/home');
+  };
+
+  handleChange = ({ target: { value, name } }) => {
+    this.setState({ [name]: value, errors: {} });
+  };
+
+  handleNext = async () => {
+    const { current, title, content, journals } = this.state;
+    try {
+      await schema.validate({ title, content }, { abortEarly: false });
+      journals.push({ title, content });
+      return this.setState({
+        current: current + 1,
+        content: '',
+        title: '',
+        errors: {},
+      });
+    } catch (error) {
+      const objError = {};
+      error.inner.forEach(fielderror => {
+        objError[fielderror.path] = fielderror.message;
+      });
+      return this.setState({ errors: objError });
+    }
+  };
+
+  handleSubmit = async () => {
+    const { title, content, journals } = this.state;
+    const { history } = this.props;
+    try {
+      await schema.validate({ title, content }, { abortEarly: false });
+      journals.push({ title, content });
+      message.success('Yes, you have added a journal');
+      history.push('/home');
+      // here, a request will be post to firebase to save data
+      // that will be as follows : [{title:'', content:'', time:'', date:'',month:''}]
+      return this.setState({ journals: [] });
+    } catch (error) {
+      const objError = {};
+      error.inner.forEach(fielderror => {
+        objError[fielderror.path] = fielderror.message;
+      });
+      return this.setState({ errors: objError });
+    }
+  };
+
+  handlePrev = () => {
+    const { current } = this.state;
+    const curr = current - 1;
+    this.setState({ current: curr, errors: {} });
```

Comment:
You can do it on the same line 
```suggestion
    this.setState({ current: current - 1, errors: {} });
```